src/lib/webpack/webpack-client-config.js

```diff
@@ -52,15 +55,15 @@ function clientConfig(env) {
 					loader: resolve(__dirname, './async-component-loader'),
 					options: {
 						name(filename) {
-							let relative = filename.replace(src, '');
-							let isRoute = filename.indexOf('/routes/') >= 0;
-
-							return isRoute ? 'route-' + relative.replace(/(^\/(routes|components\/(routes|async))\/|(\/index)?\.js$)/g, '') : false;
+							filename = normalizePath(filename);
+							if (!filename.includes('/routes/')) return false;
+							let relative = filename.replace(normalizePath(src), '');
```

Comment:
We might want to run this check on `relative` instead of `filename` - currently this will trigger a false positive if the repo is within a directory called "routes" (no clue why it would be, but still)

Reply:
✅ https://github.com/developit/preact-cli/pull/454/commits/a6608c18d4351b6a9124ed45b4d060467eb4865d